libraries/AP_Airspeed/AP_Airspeed_Backend.h

```diff
@@ -30,12 +30,15 @@
 
 class AP_Airspeed_Backend {
 public:
-    AP_Airspeed_Backend(AP_Airspeed &frontend, uint8_t instance);
+    AP_Airspeed_Backend(AP_Airspeed &frontend, class AP_Airspeed::airspeed_state &state, class AP_Airspeed_Params &params);
```

Comment:
we don't need the constructor changes at all I think, backend has access to the state and params